todo_app/app.py

```diff
@@ -1,35 +1,46 @@
-from flask import Flask, render_template, session, request, redirect, url_for
+from flask import Flask, render_template, request, redirect, url_for
 
 from todo_app.flask_config import Config
 
-from todo_app.data.session_items import get_items, add_item
+from todo_app.data.trello_items import get_to_do_items, add_item_to_list, change_list_of_item, get_board_list, list_name
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
-
 @app.route('/')
-# def index():
-#     return render_template('index.html')
 
 def index():
-    data = get_items()
-    return render_template('index.html', len = len(data), data=data )
+    get_board_list()
+    data = get_to_do_items()
+    return render_template('index.html', data=data )
 
 @app.route('/add_new_item', methods=['post'])
 def add_new_item():
 
-    item_status = request.form['item_status']
-    item_title = request.form['item_title']
-    data = get_items()
-    last_item = len(data) - 1
-    item_id = data[last_item]["id"] + 1
+    list_status = request.form['list_name']
+    item_name = request.form['item_title']
+    item_description = request.form['item_desc']
+
+    found = 0
+    for name in list_name.keys():
+        if list_status == name:
+            found = 1
```

Comment:
Minor, and this works nicely here, but note that it could lead to subtle bugs in future, for example:
* A user has just loaded your homepage
* You deploy a new version of the app into production live
* The user now submits an add item request, _before_ the `index` function has been triggered on your application
* Which means this `list_name` variable hasn't yet been populated!

If we wanted to avoid that, one approach would be to make sure the necessary list-setup happens during an initialisation phase which is explicitly called when the app starts up, rather than on someone loading the homepage